templates/python/.github/workflows/publish.yaml

```
@@ -0,0 +1,27 @@
+name: Publish
+# Template adapted from https://github.com/bakdata/ci-templates/tree/main/.github/workflows#python-poetry-release
+
+on:
+  push:
+    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
```

oh, I actually made a mistake here. The `python-poetry-release` workflow does the version bumping and release.
Therefore we should call it using a manual workflow call like this, where we select the type of release (patch, minor, major) from a dropdown.

```suggestion
on:
  workflow_dispatch:
    inputs:
      release-type:
        description: "Scope of the release; see https://python-poetry.org/docs/cli/#version"
        type: choice
        required: true
        default: patch
        options:
          - patch
          - minor
          - major
```

then you can use the `${{ github.event.inputs.release-type }}` input for the `release-type` parameter below

see https://github.com/bakdata/kpops/blob/c4499753bac1f5f6a8c4b24a8f560d7cd66dcb4c/.github/workflows/release.yaml for reference

---

That's really nice! I updated the file. I didn't find anything I needed to change when using what you suggested, is that right?

---

yep, looks correct